scripts/install_third_party_libs.py

```diff
@@ -125,9 +125,25 @@ def pip_install(package, version, install_path):
 
     # The call to python -m is used to ensure that Python and Pip versions are
     # compatible.
-    subprocess.check_call([
-        sys.executable, '-m', 'pip', 'install', '%s==%s' % (package, version),
-        '--target', install_path])
+    command = [
+        sys.executable, '-m', 'pip', 'install', '%s==%s'
+        % (package, version), '--target', install_path]
+    process = subprocess.Popen(
+        command, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    stdout, stderr = process.communicate()
+    if stderr == '':
+        python_utils.PRINT(stdout)
+    elif stderr.find('can\'t combine user with prefix') > -1:
+        python_utils.PRINT('trying by setting --user flag')
```

Comment:
Trying by setting --user and --prefix flags.

Reply:
Done!